README.md

```diff
@@ -30,34 +31,40 @@
 ](https://www.gnu.org/licenses/gpl-3.0.en.html) as a part of [HXCO](https://hxco.org)'s cooperate project.
 
  ## Features
-- Fast speed. Every static file can be distribute by [jsDelivr](https://jsdelivr.com), a global accelerated CDN.
-- Highly customizable. You can modify everything displayed on the page with no special programming technique required.
-- Straightforward configuration. You may want to create your own version, and we have provided you with convenience. All you have to do is fill in the content you want, and our code will support you and do all the other work.
+ 
+- Loading fast. Most of static file can be distributed by [jsDelivr](https://jsdelivr.com), a global accelerated open source public CDN.
+- Highly customizable. You can modify everything display on the page with no special programming skills required.
+- Easy configuration. You may want to create your own version, and we have provided you with convenience. All you have to do is fill in the content you want, and our code will support you and do all the other work.
 - "Transformer" style. Your user may open your site to pay to you in a couple of different ways. `Merger` considered everything for you and it will present itself in different ways to make sure the user's need is fitted.
 
 ## Downloads
 
-- Please use the `release` function of this project at: [https://github.com/hifocus/merger/releases](https://github.com/hifocus/merger/releases). Please only use the latest, non pre-release version of `Merger` in production environment.
+- Please use the `release` function of this project at: [https://github.com/hifocus/merger/releases](https://github.com/hifocus/merger/releases). Please only use the `latest` non `pre-release version` of `Merger` in production environment.
```

Comment:
latest and non pre-release are adjectives here. Should not be put in code blocks.